persistence/sql/identity/persister_identity.go

```diff
@@ -637,7 +639,7 @@ func (p *IdentityPersister) CreateIdentities(ctx context.Context, identities ...
 				return sqlcon.HandleError(err)
 			}
 
-			return partialErr
```

Comment:
This was the problem.

Reply:
I agree that rolling back the transaction is unwanted, but we do want to return information about which identity inserts failed. Don't we lose that information here?

Reply:
Ah sorry, I misread, looks good!